book/week_03/02-Tips-and-Tricks.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It is very early in the lecture, but you are now already well equipped to write more complex scripts and algorithms: you know (almost ;) everything you need to know about the syntax, but you might be a little lost in all the possibilities that the python language is now offering to you. *Which tool should I use to do this and that? How was this function signature again?* are very common interrogations, and it is normal to feel a bit overwhelmed.\n",
+    "It is very early in the lecture, but you are now already well equipped to write more complex scripts and algorithms: you know (almost ;)) everything you need to know about the syntax, but you might be a little lost in all the possibilities that the python language is now offering to you. *Which tool should I use to do this and that? How was this function signature again?* are very common interrogations, and it is normal to feel a bit overwhelmed.\n",
```

Comment:
Haha you are ready for this conference then ;-) 

https://xkcd.com/541/

Reply:
Haha [better not :)]